plaso/parsers/__init__.py

```diff
@@ -45,6 +45,7 @@
 from plaso.parsers import plist
 from plaso.parsers import popcontest
 from plaso.parsers import pls_recall
+from plaso.parsers import ps_transcript
```

Comment:
We should rename 'ps' to 'powershell' to remove any ambiguity.
Maybe powershell_transcript ?